cves/kernel/CVE-2017-5576.yml

```diff
@@ -356,8 +360,10 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+   Checks the roundup() for the shader_rec_offset against bin_cl_size to
```

Comment:
Perhaps adding something to the beginning "Involved adding a Check in roudup() for..." would make the sentence more clear as an answer to the question